ДЗ JS

kargamant opened the pull request.

kargamant committed:
Update and rename hw01_normal.py to ДЗ урок 1 normal

kargamant committed:
Update ДЗ урок 1 normal

kargamant committed:
Update ДЗ урок 1 normal

kargamant committed:
Update hw01_easy.py

kargamant committed:
Update hw01_easy.py

kargamant committed:
Update hw02_easy.py

kargamant committed:
Update hw02_normal.py

kargamant committed:
Update hw03_easy.py

kargamant committed:
Update hw03_normal.py

kargamant committed:
Update hw03_easy.py

kargamant committed:
Update hw04_easy.py

kargamant committed:
Update hw04_normal.py

kargamant committed:
Update hw04_normal.py

kargamant committed:
Update hw05_easy.py

kargamant committed:
Update hw05_normal.py

kargamant committed:
Update hw05_hard.py

GefMar commented on lesson05/home_work/hw05_normal.py:
```diff
@@ -10,6 +14,38 @@
 
+print('Привет! Я утилита')
+print('Вот что я умею')
+print('1. Перейти в папку\n2. Просмотреть содержимое текущей папки\n3. Удалить папку\n4. Создать папку\n')
```
А можно было сделать на основании словаря и структуры

kargamant replied:
Знаю, но у меня возникла проблема. Я создал словарь, а там все ключи вызывались подряд, как бы я их не пытался вызывать по отдельности. Попытался решить эту проблему с пмощью цикла фор, не помогло.

kargamant committed:
Update hw06_easy.py

kargamant committed:
Update hw06_normal.py

kargamant committed:
Create PyGAME1

kargamant committed:
Create PYGAME ULTIMATE

kargamant committed:
Максимум что я смог сделать

основная запара была с картой, в итоге так и не успел полностью доделать игру, но обязательно продолжу на следующих выходных(уже скорее для себя, потому что я не успею сдать)

kargamant committed:
Create JS дз урок 2

kargamant committed:
Create ДЗ js урок 3

kargamant committed:
Create JS дз урок 3

kargamant committed:
Create JS дз урок 3

kargamant committed:
Rename JS дз урок 3 to JS дз урок 4

kargamant committed:
Merge pull request #1 from kargamant/JS-дз

JS ДЗ

kargamant committed:
Create JS дз урок 5
